portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.data;
+
+import java.util.*;
```

Comment:
I don't think you need this import. String is automatically imported (as is everything in java.lang.*)